src/components/Profile/ProfileEdit.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
```

Comment:
you don't seem to be using this bit.  it looks like you're still working out your strategy maybe?